impeller/renderer/compute_unittests.cc

```diff
@@ -308,6 +308,7 @@ TEST_P(ComputeTest, MultiStageInputAndOutput) {
     CS1::BindOutput(*pass, DeviceBuffer::AsBufferView(output_buffer_1));
 
     ASSERT_TRUE(pass->Compute(ISize(512, 1)).ok());
+    pass->AddBufferMemoryBarrier();
```

Comment:
Since we're now running concurrently, we need a memory barrier here so that step 2 sees the output of step 1.